core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -204,7 +226,7 @@ private void connectToChunkList(List<Chunk<K, V>> engaged, List<Chunk<K, V>> chi
                 break;
             }
             // chunk is not in list (someone else already updated list), so we're done with this part
-            if ((curr == null) || (prev == null)) {
+            if (curr == null) {
                 //TODO Never reached
```

Comment:
If already making all those changes. May be to remove this comment as well. If it would be so, ItelliJ would show it for sure...

Reply:
removed comment